CHANGES.md

```diff
@@ -1,3 +1,22 @@
+## edge-19.7.4
+
+* CLI
+  * Made the `linkerd routes` command traffic-split aware; consider resources
+    which are members of a leaf service with non-zero weight to be members of
+    the apex service for the purpose of determining which routes to display
+  * Added pod status to the output of the `linkerd stat`command (thanks
+    @jonathanbeber!)
+  * Fixed `linkerd stat` wrong "meshed" count when resources share the same
+    label selector for pods (thanks @jonathanbeber!)
+  * Improved output of the `linkerd edges` command and added a new `-o wide`
```

Comment:
to be clear, the previous default behavior was to show the identity and not namespaces. now the namespace of the source and the destination are shown by default, and you can use `-o wide` to show identity. I might rephrase as:

```
Added namespace information to the `linkerd edges` command output and a new `-o wide` flag that shows the identity of the client and server if known
```